.github/workflows/ci.yml

```diff
@@ -120,49 +120,26 @@ jobs:
         run: |
           sudo mkdir /stdb
           sudo chmod 777 /stdb
+
```

Comment:
Why did this file change? Is that related to this PR?

Reply:
I was trying to test this and wanted to regenerate client SDK as part of the CI. That backfired - cross-repo testing is still pretty damn hard - so the test will have to be as part of client SDK PR, so I reverted the `spacetime generate` changes but left this one as it's purely for clarity. I'm happy to split it out as it's not related anymore.

Reply:
I'm fine to keep it in as long as you add a note about it to the PR description.

Reply:
Done